src/expr/node_manager.h

```diff
@@ -444,6 +444,9 @@ class NodeManager {
   /** Get a Kind from an operator expression */
   static inline Kind operatorToKind(TNode n);
 
+  /** Get corresponding application kindfor function */
+  static inline Kind getKindForFunction(TNode fun);
```

Comment:
We don't use `inline` anymore. This is assumed to be managed by the compiler.

Reply:
Missed that memo. :) Will fix.